pkgs/data/fonts/cozette/default.nix

```diff
@@ -9,16 +9,22 @@ stdenvNoCC.mkDerivation rec {
     hash = "sha256-v1UWrVx1PnNPiFtMMy4kOkIe//iHxx0LOA4nHo95Zws=";
   };
 
+  buildInputs = [ psftools ];
+
   installPhase = ''
     runHook preInstall
 
     install -Dm644 *.ttf -t $out/share/fonts/truetype
     install -Dm644 *.otf -t $out/share/fonts/opentype
     install -Dm644 *.bdf -t $out/share/fonts/misc
     install -Dm644 *.otb -t $out/share/fonts/misc
+    install -Dm644 *.fnt -t $out/share/fonts/misc
```

Comment:
https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=psf-cozette looks like using bdf2psf for this purpose. Will fnt2psf be better replacement?

Reply:
As far as I know, `.fnt` is provided for Windows. So I don't have confidence having that in nixpkgs will be good.